opengrok-tools/src/main/python/opengrok_tools/version.py

```diff
@@ -0,0 +1,15 @@
+def get_version(version):
```

Comment:
add a top-level comment that the file is being modified by the Maven build

Reply:
done